crypto/store/store_lib.c

```diff
@@ -513,7 +513,14 @@ static int ossl_store_close_it(OSSL_STORE_CTX *ctx)
         ret = ctx->loader->p_close(ctx->loader_ctx);
 #ifndef OPENSSL_NO_DEPRECATED_3_0
     if (ctx->fetched_loader == NULL)
+# if defined (OPENSSL_TANDEM_FLOSS) && defined (_SPT_MODEL_)
+   /* NonStop SPT redirects close to spt_close causing a compile failure. */
+#  undef close
+# endif
         ret = ctx->loader->close(ctx->loader_ctx);
+# if defined (OPENSSL_TANDEM_FLOSS) && defined (_SPT_MODEL_)
```

Comment:
There is no need for this at all.  `close(2)` is never calls.

Reply:
> 
> 
> There is no need for this at all. `close(2)` is never called here.

The textual `close` is replaced by the c99 preprocessor with `spt_close` by virtue of matching the macro's structure.

Reply:
Field has been renamed to `closefn`, so this change goes away.